var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -213,6 +219,7 @@ class Paraview(CMakePackage, CudaPackage):
     depends_on("xz")
     depends_on("zlib")
     depends_on("libcatalyst", when="+libcatalyst")
+    depends_on("hip", when="+hip")
```

Comment:
I think this will require a minimum version, ref VTK-m

Reply:
VTK/ParaView don't support external VTK-m, so how would it ever be out-of-sync?

Reply:
it requires a minimum HIP version, `depends_on("hip@3.7:", when="+rocm")`

Reply:
although, I would say make it a minimum of 5.2, I know we have had issues with earlier versions of HIP and 5.2 is the only version the ParaView has actually been tested against.

Reply:
> 4.5 < 5.1 is doable with -O0 optimization levels, anything beyond -O0 will hang template instantiations due to a known bug in hipcc

Reply:
Minimum of 5.2.0 enforced.